packages/loopring_v3/contracts/iface/ILoopringV3.sol

```diff
@@ -141,13 +141,13 @@ abstract contract ILoopringV3 is ILoopring
     /// @param onchainDataAvailability True if the exchange has on-chain
     ///        data-availability, else false
     /// @return True if the exchange has staked enough, else false
-    function canExchangeCommitBlocks(
+    function canExchangeSubmitBlocks(
```

Comment:
Okay for me, but I kept it like this so that 3.5 exchanges are compatible with the existing Loopring contracts.

Reply:
I get it. Will revert this back.